labs/fabric-chaincode-wasm.md

```diff
@@ -0,0 +1,26 @@
+# Lab Name
+Hyperledger Fabric WebAssembly chaincode
+
+# Short Description
+The project aims to integrate WebAssembly in hyperledger fabric, which will give chaincode developers a wider variety of languages to choose from when constructing their smart contracts.
```

Comment:
Capitalize Hyperledger Fabric.

Reply:
Done

Reply:
I was actually talking about the "hyperledger fabric" on line 5. Can you revert the above change and fix on line 5 to reflect "Hyperledger Fabric".

Reply:
Oh sorry, I misunderstood. Fixed it now